releases-beta/how-to-guides/resolving-a-cherry-pick-failure.md

```diff
@@ -0,0 +1,41 @@
+# Resolving a Cherry-Pick Failure
+
+This guide walks you through the steps to handle a failed cherry-pick during a release process in Aviator. Cherry-picking allows you to selectively integrate specific changes (commits) into a base release. If a conflict occurs, you can resolve it manually.
+
+Setup a repository, connect it to Aviator and [create a release project](./creating-a-release-project.md) for this guide.
+
+## Create a Base Release
```

Comment:
We should not need these steps. We are explaining a scenario of how to handle a Cherry pick, we do not need to explain how to create a cherrypick

Reply:
Actually I discussed this with @draftcode and he suggested that we should keep these instructions. So please ignore my comment here.